### ISSUE TITLE
{{ nav from="/{segment_1}" }} no output

### ISSUE THREAD

**Issue by Pfeifakopf** [statamic/cms#1542]
This error is with a multi-site, don't know, if it works on single sites.
```
   {{ nav from="/{segment_1}" max_depth="2" }}
      <li>
         <a href="{{ url }}"{{ if is_current || is_parent }} class="on"{{ /if }}>{{ title }}</a>
         {{ if is_current || is_parent }}
            {{ if children }}
               <ul>{{ *recursive children* }}</ul>
            {{ /if }}
         {{ /if }}
      </li>
   {{ /nav }}
```
```
Facade\Ignition\Exceptions\ViewException
Call to a member function pages() on null
```

**Comment by jasonvarga** [comment]
I can't trigger any error. Can you please provide more details about your setup?

**Comment by Pfeifakopf** [comment]
I have 2 navigation collections, pages and products. If I'm on the products collection, this error shows up. As long I'm on the pages collection, all works fine. With {{ nav from="{ parent:uri }" }} it's the same.

**Comment by scottzirkel** [comment]
I'm having the same issue. Even tried copying the example straight from the docs like above. Looks like the `$entry->id` is null in `Structures\TreeBuilder`

The navigation works fine if I hard code it into the nav (`{{ nav:collection_name }}`), but I want to keep it dynamic, since I have a few different navs setup.

**Comment by jasonvarga** [comment]
I think this issue is solved by 5a9d7a06. Feel free to reopen if not.

You can also now just do `:from="segment_1"`.
